src/admin.rs

```diff
@@ -150,7 +150,7 @@ impl FluvioAdminJS {
 
     #[node_bindgen]
     async fn list_spu(&mut self) -> Result<ArrayBuffer, FluvioErrorJS> {
-        serde::__private::Ok(self.js_list::<SpuSpec>().await?)
```

Comment:
wat? I didn't even know this was in here.

Reply:
I have seen this in the past suggested by code editor autocompletion

Reply:
I've seen it in come build errors in very rare scenarios. I've fixed them with a `cargo clean` and a `cargo build`.